recipes/jinja2/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version="2.8" %}
+
+package:
+  name: jinja2
+  version: {{ version }}
+
+source:
+  fn: Jinja2-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/J/Jinja2/Jinja2-{{ version }}.tar.gz
+  md5: edb51693fe22c53cee5403775c71a99e
+
+build:
+  number:  0
+  script: python setup.py install --old-and-unmanageable
```

Comment:
The use of `python` I believe is just to ensure the correct environment is used for installation. So we could probably just use `pip install --no-deps .`. Not sure if there are other arguments needed or not.